osbs/utils.py

```diff
@@ -613,7 +615,8 @@ class Labels(object):
         LABEL_TYPE_RUN: ('run', 'RUN'),
         LABEL_TYPE_INSTALL: ('install', 'INSTALL'),
         LABEL_TYPE_UNINSTALL: ('uninstall', 'UNINSTALL'),
-        LABEL_TYPE_OPERATOR_MANIFESTS: ('com.redhat.delivery.appregistry',)
+        LABEL_TYPE_OPERATOR_MANIFESTS: ('com.redhat.delivery.appregistry',),
```

Comment:
I haven't updated this name to `LABEL_TYPE_OPERATOR_APPREGISTRY_MANIFESTS` to keep BW compat. Let me know if you disagree and I should create alias to old manifest constant instead.